root/etc/e-smith/events/actions/nethserver-dc-user-modify

```diff
@@ -49,25 +49,33 @@ if ($? != 0) {
 
 #Create a temporary ldif file (root:root 0600)
 my $fileName = File::Temp->new(
-    UNLINK   => 0,
+    UNLINK   => 1,
     TEMPLATE => "$userName.XXXXXXXXXX",
     DIR => '/var/lib/machines/nsdc/var/spool/'
 );
 
-#create the ldif content
+#we need the relative path in the container
+my $relativePathFile = $fileName; 
+$relativePathFile =~ s|/var/lib/machines/nsdc||;
+
+#retrieve the dn from samaccountname
+my $command = qq(systemd-run -M nsdc -q -t /usr/bin/ldbsearch -H /var/lib/samba/private/sam.ldb "samaccountname=$userName" dn | sed -n '/^dn: / { s/\r// ; p ; q }');
+open DN, "$command|";
+my $dn = <DN>;
```

Comment:
Can the three lines above be grouped in a single `qx()` (backtick operator) expression?